guide/coaching.md

```diff
@@ -40,42 +44,59 @@ permalink: /guide/coaching/
 
 <p>There are a lot of good reasons to become a coach. First and foremost, you will be giving back to the community for a great cause: by empowering women to contribute to open source. It's a great way to share your knowledge. We also believe that coaching can be really rewarding for your own work, that it can improve the way you write code and help you grow as a developer, as well as make you think about concepts you take for granted. And above all, we promise you an overwhelming buzz looking into the happy faces of your students when they succeed.</p>
 
-<h3 id="what">What does coaching require?</h3>
+<h3 id="remote">Can I coach remotely?</h3>
+
+<p>The short answer is:</p>
+
+* Yes
+
+
+<p>We highly recommend students and coaches to spend their summer of code in one location, ideally sharing one working space. We believe that the ability to sit together in front of the screen can make a more beneficial and fruitful tandem. However, sometimes it’s not possible, and then remote coaching comes as a good solution.</p>
+
+<h3 id="what">What will I be doing as a coach?</h3>
 
 * Going over the basics if needed
 * Guiding students through coding steps
 * Showing how to debug and troubleshoot
 * Showing students the way to a solution
 
-<p>It is important for coaches to teach the process of programming - showing the way to a solution rather than telling the answer. This will sometimes include going over the basics, showing how to debug and troubleshoot, or discussing concepts on paper. Coaches should spend time with their students in front of the screen, guiding them through the relevant coding steps (eg. by asking mindful questions). Every project is different; just make sure your students really understand what you are trying to teach them, at the risk of not being <abbr title="Don't Repeat Yourself">DRY</abbr> :)</p>
+
+<p>It is important for coaches to teach the process of programming - showing the way to a solution rather than telling the answer. This will sometimes include going over the basics, showing how to debug and troubleshoot, or discussing concepts on paper. Coaches should spend time with their students, guiding them through the relevant coding steps (eg. by asking mindful questions). Every project is different; just make sure your students really understand what you are trying to teach them, at the risk of not being <abbr title="Don't Repeat Yourself">DRY</abbr> :)</p>
 
 <p>While we know that sometimes life gets in the way and that unexpected events happen: We really encourage all coaches supporting a team to try their hardest to commit, and not to drop out of the program halfway through. <strong>Your support is essential for teams to succeed</strong>!</p>
 
+<h3 id="expertise">Should I have a good knowledge of a project chosen by my team?</h3>
```

Comment:
change to “Should I have in-depth knowledge of the project(s) chosen by my team?”